### DISCUSSION TITLE
How to verify whisper doesn't sent data outside to OpenAI nor anywhere else?

### DISCUSSION THREAD

**Discussion by shendelin** [openai/whisper#2437] [Q&A]
I'm building a prototype for transcribing some sensitive data. To get the prototype approved, I will have to show that it is fully running locally, and no data is sent outside to OpenAI nor anywhere else. Any recommendations how to prove it?

**Comment by 0mjs** [comment]
whisper doesn't use openai apis you can just run it locally.

run your code in the terminal and disable the network

**Comment by shendelin** [comment]
Thanks for the tips!
